2_segmenting_data/README.md

```diff
@@ -0,0 +1,104 @@
+# Segment Schwann Cells from NF1 Data
+
+In this module, I present the pipeline for segmenting nuclei and cytoplasm from the NF1 pilot data.
+
+## Segmentation
+
+The method used for segmentation for both the nuclei and cytoplasm is an algorithm called [CellPose](https://doi.org/10.1038/s41592-020-01018-x). Through the Python implementation of this method, this pipeline can be reproducible and is easy to manipulate for various datasets.
+
+## Nuclei Segmentation
+
+After experimenting with the various parameters in CellPose on all of the pilot NF1 DAPI (nuclei) channel images, I settled on the following parameters for CellPose nuclei segmentation:
+- `model_type : "cyto"` This parameter sets the model for CellPose to use as the cytoplasm model, which I found segments nuclei better than the established nucleus model. 
+More information about the various CellPose models can be found at https://cellpose.readthedocs.io/en/latest/models.html.
+- `channels : [0,0]` This parameter sets the channels to be used by the model to segment, which in the case of nuclei, we do not ned to set channels because the images are greyscale.
+- `diameter : 50` This parameter sets the cell diameter for cells within the image to an extact value. 
+This can also be calculated using the `calculate` button in the CellPose GUI. 
+I found that the segmentation worked better when the diameter was established with a set value. 
+- `flow_threshold : 0` This paramenter decreases the maximum allowed error of the flows for each mask (default is `flow_threshold : 0.4`).
+- `remove_edge_masks : True` This parameter removes any masks from CellPose that are touching an edge of the image.
+
+To show how I determined the best parameters for nuclei segmentation, below is a figure showing two rows of nuclei, with an image of the nuclei without segmentation (left), a segmented nuclei using non-optimal parameters (middle), and a properly segmented nuclei (right). 
```

Comment:
It might be worth adding details here about what _non-optimal_ parameters are. Do you have insight into which parameter was key to changing?

Reply:
The one parameter that seemed to make a difference in trying to find a "bad set of parameters" was changing the cell diameter to extremes (too small or too big). So I can add a part in here talking about how important the cell diameter portion of the parameters is for proper segmentation.

Reply:
This part has been added!